### ISSUE TITLE
TSC Meeting (6 Oct 2020): OSS Economy, StackStorm Partners, TSC Chair Election, v3.3.0 Release

### ISSUE THREAD

**Issue by arm4b** [StackStorm/discussions#50]
Another periodic @StackStorm/tsc 1 hour meeting will take place on Tuesday, `6th Oct 2020, 09:30 AM US Pacific`.
See #33 for more info about how to join.

## Meeting Agenda

- PackageCloud, Expenses and Extreme Networks (@m4dcoder)
  - See [StackStorm Expenses](https://github.com/StackStorm/discussions/issues/36) for more context
- StackStorm Partners #51
  - Beta of https://stackstorm.com/partners/
  - Recommended ways to Market and Advertise for the Partners
  - Conflict of interest situations https://github.com/StackStorm/st2/pull/5045
- TSC Chair Election (raised by @dzimine)
  - Discussion about the terms, conditions, mechanism of electing the TSC Chair
  - [Governance.md](https://github.com/StackStorm/st2/blob/master/GOVERNANCE.md#maintainer-roles) will need a change
- `v3.3.0` Release Preparation
  - Release Manager @nmaludy, Release Assistant @blag
  - Pre-release Testing
  - Release Management and Automation
- Release Cadence
- Website action required: Netflix requested logo removal - so far unofficially (@dzimine )
- Follow-up: ChatOps #8 (@blag)
- Follow-up: Python 2 Deprecation #40 

cc @StackStorm/maintainers @StackStorm/contributors.

## Attendees
Total `~15` attendees were present
- Host: @armab 
- @StackStorm/maintainers (7/10): @dzimine @m4dcoder @blag @armab @punkrokk @amanda11 @nmaludy 
- @StackStorm/contributors: @winem, @carlos, @emptywee

**Comment by vivekorch** [comment]
Thank you. I will join the meeting on Oct 6th.

**Comment by dzimine** [comment]
I'll add a bullet on Netflix request to withdraw use of their logo.

**Comment by dzimine** [comment]
# Notes an Action items from the meeting

> Note: capturing entire conversation is not a goal here, come and participate not to miss it!


### 1. PackageCloud, Expenses and Extreme Networks ( [@m4dcoder](https://github.com/m4dcoder) )
* See  [StackStorm Expenses](https://github.com/StackStorm/discussions/issues/36)  for more context

Immediate: 
   - [ ] @armab @m4dcoder please capture your decision on PackageCloud and follow through 
   - [ ] @m4d @jp pls handle AWS per your conversation to use up credits JP secured

Going forward:
  * We will be using community money to pay the expenses. But we realize we don't have a sustainable source of funding, and need ideas for a solution(s).
### 2. StackStorm Partners, commercial interests, and conflict of interest situation  [#51](https://github.com/StackStorm/discussions/issues/51)
We may continue to disagree on OSS philosophy, and perils & failures of regulations (fair amount of time spent on it and it’s good to flash it out once in a while).  But let’s focus on identifying and solving immediate problems. We identified: 

* problem: community members abuse Slack and forum to plug their commercial offerings.
	* solution: Slack and Forum code of conduct, including “non-soliciting” are defined and hosted, pointed to new community members as part of subscription to Slack, Forum.
	- [ ] action: define code of conduct by taking a standard one, host (DZ idea, better ideas most welcome: host as a post on Forum, link from “welcome to Community” email or from “Welcome to community” invitation popup. 
	- [ ] owner: maybe @armab with @amanda11 help (per your spot-on slack input)? 
* problem: Some use of email lists collected from community members may violate privacy, as defined by LF. The two sources of emails are slack and ActiveCampaign (which we need to send OpenSource communication). Th
	* solution: hide email addresses in Slack (done). User agrees to receive emails from ST2 community, and has an option to unsubscribe (in place). Regulate the use of Active Campaign emails - (DZ-thinks: emails send on behalf of OS by TSC members, TSC promotes partners’s list, and partners’ good deeds for community, partners won’t be allowed to use the email list for their campaigns). Prosecute violators who abuse admin rights to steal these emails - feather and tar, send to Siberia, cancel (pending)
	- [ ] action: draft the AC use regulation, 1/2 page, discuss, vote.
	- [ ]  owner: TBD - maybe @punkrokk ?
* **problem:** use of info@ sales@ stackstorm.com 
	* **solution:** automatic response with a reference to a partner page.  No soliciting (it’s not OK for a user to receive follow-up emails from a partner).
	- [ ]  **action:** set up automatic email response, when partner page launched, update it with a reference. 
	- [ ]  **owner**: TBD 
* **problem:** launching of the [https://stackstorm.com/partners/](https://stackstorm.com/partners/) is blocked by some unresolved details
	* solution [JfDI](https://www.goodreads.com/book/show/3159176-jfdi-just-do-it) - **separate to-do needed**
	- [ ]  **owner:** TBD, @dzimine offers help.

Let’s continue to identify the problems and knocking them down.

Also: not discussed in full, saved for later: Partner’s economy, the TSC role in providing value for partners; @arma’s proposal of mandatory donations 
	
### 3. TSC Chair Election (raised by  [@dzimine](https://github.com/dzimine) )

- [x] DZ to express the process as a PR to Governance.md: role definition (model - @arma in the last 6 months), term (6 months), election process. -> between now and next TSC meeting.
    * done, https://github.com/StackStorm/st2/pull/5071 

### 4. v3.3.0 Release Preparation

* Not discussed in details, manly taken off-line.

### 5. Release Cadence

* DECIDED: switch to releasing every 3 months (fixed time), with all the features 

### 6. Website action required: Netflix requested logo removal

Not discussed at the meeting but the action is obvious, we have to do it while there is still good will. 

- [ ] remove the logo from two places at the homepage - @dzimine - by Oct 24 (no particular reason for a day but other than having a time on commitment).
